contracts/interfaces/IDCAStrategies.sol

```diff
@@ -469,6 +488,9 @@ interface IDCAStrategies is IDCAStrategiesManagementHandler, IDCAStrategiesPermi
   /// @notice Thrown when a user provides a zero address when they shouldn't
   error ZeroAddress();
 
+  /// @notice Thrown when a user tries to perform an action for a token they do not own
+  error NotOwner();
```

Comment:
I think that since we are going with the permission error instead of this error, we can move this back to under `IDCAStrategiesPermissionsHandler`